collect_app/src/main/java/org/odk/collect/android/activities/GeoTraceActivity.java

```diff
@@ -495,6 +544,34 @@ private void updateUi() {
         playButton.setEnabled(location != null);
         backspaceButton.setEnabled(numPoints > 0);
         clearButton.setEnabled(!recordingActive && numPoints > 0);
+
+        // Trace settings dialog
+        radioGroup.check(recordingMode == AUTOMATIC_RECORDING ? R.id.trace_automatic : R.id.trace_manual);
+        autoOptions.setVisibility((recordingMode == AUTOMATIC_RECORDING) ? View.VISIBLE : View.GONE);
+        autoInterval.setSelection(intervalIndex);
+        accuracyThreshold.setSelection(accuracyThresholdIndex);
+
+        // GPS status
+        boolean usingThreshold = isAccuracyThresholdActive();
+        boolean acceptable = location != null && isLocationAcceptable(location);
+        int intervalSeconds = INTERVAL_OPTIONS[intervalIndex];
+        int thresholdMeters = ACCURACY_THRESHOLD_OPTIONS[accuracyThresholdIndex];
+        locationStatus.setText(
+            location == null ? getString(R.string.geotrace_location_status_searching) :
+            !usingThreshold ? getString(R.string.geotrace_location_status_accuracy, location.sd) :
+            acceptable ? getString(R.string.geotrace_location_status_acceptable, location.sd) :
+                getString(R.string.geotrace_location_status_unacceptable, location.sd)
```

Comment:
For readability can you move the colons to the beginning of the line and indent e.g.

```java
test1 ? a
  : test2 ? b
  : test3 ? c
```

(because the colon can be read as "else if" so it should begin a statement rather than ending one)

Reply:
Done!